src/main/java/edu/harvard/dbmi/avillach/dictionary/dashboarddrawer/DashboardDrawerList.java

```diff
@@ -0,0 +1,6 @@
+package edu.harvard.dbmi.avillach.dictionary.dashboarddrawer;
+
+import java.util.List;
+
+public record DashboardDrawerList(List<DashboardDrawer> dashboardDrawerList) {
```

Comment:
You made yourself a note to get rid of this record and just return a list. I agree with that note.

Reply:
I made a comment that DashboardDrawerRecord was redundant as it's already a record. 

DashboardDrawerList should be a list of DashboardDrawer as that is the collection type of this record class.    

DashboardDrawer is just a record.